src/ui/SmartSnippet/SmartSnippet.ts

```diff
@@ -60,19 +68,45 @@ export class SmartSnippet extends Component {
   }
 
   public createDom() {
-    this.buildShadow();
-    this.buildSourceContainer();
+    this.element.appendChild(this.buildAnswerContainer());
+    this.element.appendChild(
+      new UserFeedbackBanner(
+        isUseful =>
+          this.usageAnalytics.logCustomEvent<IAnalyticsNoMeta>(
+            isUseful ? analyticsActionCauseList.likeSmartSnippet : analyticsActionCauseList.dislikeSmartSnippet,
+            {},
+            this.element,
+            this.lastRenderedResult
+          ),
+        explaination => console.info(`Explaination: ${explaination}`)
+      ).build()
+    );
+  }
+
+  private buildAnswerContainer() {
+    return (this.answerContainer = $$(
```

Comment:
The `this.answerContainer` seems to be only assigned to, but never read from. Could we remove it ?

Reply:
Woops, I forgot to remove it. Thank you.